todo-app/src/todoItems.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import TodoDescription from './todoDescription'
+import TodoState from './todoState'
+
+class todo extends Component {
+    render() {
+        return (
+            <div className='todoList' className='todoState' className='Description'>
+                <ul>
```

Comment:
You haven't fixed the problem that for each todo item, we now render both `ul` and `li` elements.